src/kudu/consensus/leader_election.cc

```diff
@@ -67,6 +68,11 @@ DEFINE_bool(trust_last_leader_entries, true,
 
 namespace kudu {
 namespace consensus {
+
+using std::string;
```

Comment:
nit: I would usually prefer to not shorten `string` and just use std::string everywhere since string is a pretty common name.